### PR TITLE
Feature 15 add latexmkrc file

### LINKED ISSUE #15: Add latexmkrc file for latexmk usage
Suggestion to add a `.latexmkrc` makefile to allow usage of `latexmk` compilation tool

### PR DESCRIPTION
**Pull request by kahn-jms**
Yeet

### CODE REVIEW

File: .latexmkrc [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+$pdflatex = "xelatex -interaction=nonstopmode -file-line-error --synctex=1 -shell-escape %O %S";
+
+# Need to turn of ps and dvi generation for lualatex/xelatex, if you want these you'll 
+# need to convert the pdf later.
+$pdf_mode = 1;
+$postscript_mode = 0;
+$dvi_mode = 0;
+
+# Put all generated files into a build dir to keep things a bit cleaner
+#$aux_dir = "./build";
```
Comment (Markus-Goetz) [inline .latexmkrc]: This is not what the Makefile does. Can we be consistent here>

File: .latexmkrc [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+# Increase max repeats of luatex/xelatex runs, mostly needed for glossaries
+#$max_repeat = 9;
```
Comment (Markus-Goetz) [inline .latexmkrc]: The Makefile also only does 2 runs

**Review by Markus-Goetz — CHANGES_REQUESTED** [review]

File: .latexmkrc (hunk 1 above)
Comment (kahn-jms) [reply .latexmkrc]: Not sure I understand, this is commented out but left as an example if the users want to compile into a `build/` directory. I can remove it entirely though?

File: .latexmkrc (hunk 2 above)
Comment (kahn-jms) [reply .latexmkrc]: Also commented out, the latexmk default is 5 but I can change it to 2?

File: .latexmkrc (hunk 1 above)
Comment (Markus-Goetz) [reply .latexmkrc]: remove altogether

File: .latexmkrc (hunk 2 above)
Comment (Markus-Goetz) [reply .latexmkrc]: 2 would be good with makefile consistency

File: .latexmkrc (hunk 1 above)
Comment (kahn-jms) [reply .latexmkrc]: Removed

**Review by Markus-Goetz — APPROVED** [review]

**Pull request merged**